pip-287/src/QuestionCard.js

```diff
@@ -0,0 +1,210 @@
+        </MDBContainer>
+      )
+
+    }
+  }
+  render() {
+
+    return (
+      <Draggable draggableId={this.props.parentKey + 1} index={this.props.index}>
+        {(provided) => (
+          <div ref={provided.innerRef}
+            {...provided.draggableProps}>
+            <MDBContainer className="block-example border pt-4 pb-2 mt-2 ml-0 mr-0">
+              <div className="row">
+                <div className="col-2 align-self-center">
+                  <MDBCard className="text-center " >
+                    <MDBListGroup className="h-100 PointerMouse">
+                      <MDBListGroupItem onClick={this.MoveUp}>
+                        <MDBIcon icon="angle-up " />
+                      </MDBListGroupItem>
+                      <MDBListGroupItem  {...provided.dragHandleProps}>
+                        <MDBIcon icon="ellipsis-h" />
+                      </MDBListGroupItem>
+                      <MDBListGroupItem onClick={this.DeleteAnswer}>
+                        <MDBIcon icon="trash-alt" />
+                      </MDBListGroupItem>
+                      <MDBListGroupItem onClick={this.MoveDown} >
+                        <MDBIcon icon="angle-down" />
+                      </MDBListGroupItem>
+                    </MDBListGroup>
+                  </MDBCard>
+                </div>
+                <div className="col">
+                  <MDBInput type="textarea" value={this.props.data.questionText} onChange={this.questionTextChange} label="Pytanie" rows="2" />
+                  <div>
+                    <MDBInput type="textarea" value={this.props.data.questionDescription} onChange={this.questionDescriptionChange} label="Opis pytania" rows="2" />
+                  </div>
+                </div>
+              </div>
+              <br />
+              <div className="d-flex justify-content-between align-items-end">
+                <p className="d-flex " style={{ color: "#757575" }}>Odpowiedź</p>
+                <div className="d-flex align-items-end">
+                  <DropdownButton id="dropdown-basic-button" onSelect={this.questionTypeChange} title={this.MapQuestionType(this.props.data.questionType)}  >
+                    <Dropdown.Item eventKey='shortText'>Short</Dropdown.Item>
+                    <Dropdown.Item eventKey='longText'>Long</Dropdown.Item>
+                    <Dropdown.Item eventKey='singleOption'>Radio</Dropdown.Item>
+                    <Dropdown.Item eventKey='multipleOptions'>Check</Dropdown.Item>
+                  </DropdownButton>
+                </div>
+              </div>
+              {
+                this.GetQuestionRender()
+              }
+              <div className="d-flex flex-row-reverse mt-3">
+                <div className="custom-control custom-checkbox ">
+                  {/* <input checked={this.state.isRequiered} onChange={this.isRequieredChange} type="checkbox" class="custom-control-input" id="IsRequiredCheckbox" />
```

Comment:
Co to ?

Reply:
Style dla checkboxa, zostaną poprawione na reactowo-boostrapową wersje

Reply:
Głównie mi chodziło o ten zakomentowany kawałek kodu :P

Reply:
Stary nie używany kod, już został usunięty